classes/ThemeScanner.php

```diff
@@ -55,12 +55,18 @@ public function scanThemeConfigForMessages()
         $keys = [];
 
         foreach ($config as $locale => $messages) {
+            if( is_string($messages) ) {
+                $messages = $theme->getConfigArray('translate.'.$locale);
+            }
             $keys = array_merge($keys, array_keys($messages));
         }
 
         Message::importMessages($keys);
 
         foreach ($config as $locale => $messages) {
+            if( is_string($messages) ) {
```

Comment:
Fix spaces please (one of these days I'll implement a style bot that does these comments for me :) )

`if (is_string($messages)) {`